roles/vault/defaults/main.yml

```diff
@@ -87,4 +87,4 @@ vault_temp_config:
       address: "0.0.0.0:{{ vault_port }}"
       tls_disable: "true"
 vault_temp_container_name: vault-temp
-vault_version: 0.6.4
+vault_version: 0.7.3
```

Comment:
it's require change [binary checksum](https://github.com/kubernetes-incubator/kubespray/pull/1405/files#diff-6142088e026c4d7163f2f8865f3bf5e0R11)